Changelog

```diff
@@ -1,3 +1,60 @@
+    - FEATURE: Add (experimental) I2P support (glassez) cdded6cef7710fb25093ea142edea3d3b7479447
+    - FEATURE: Provide UI Theme editor (glassez) 77aa85fbd3af12b319f8f0b8372f4ba186d10c80
+    - FEATURE: Various UI theming improvements (glassez)
+    - FEATURE: Implement torrent tags editing dialog (glassez) d40be79c6933fd911644b0b0c833893ea480dc90
+    - FEATURE: Implement "Add torrent params" editing widget (glassez) 0cc29f18510b084b131fd0b02963a7aff909de01
+    - FEATURE: Revamp "Watched folder options" and "Automated RSS downloader" dialog (glassez) 93a1e58554fe9beff657b1302cc358722e8b0a1b 905f1416572913cf0bb0d143b7da381cd544fad8
+    - FEATURE: Allow to use another icons in dark mode (glassez) 72ac92ec682b96924bdec606bc4383430f3f7067
+    - FEATURE: Allow to add new torrents to queue top (glassez) 5d4766edbe88c768e8fa9a91ca3dadbf9a59aedf
+    - FEATURE: Allow to filter torrent list by save path (Tom) 0bb0829a9a22e4b468af6bcbc119df5d1d1571ec
+    - FEATURE: Expose 'socket send/receive buffer size' options (Chocobo1) 77411760a093bffafa3ae2c4f139990cd4ff55e6
+    - FEATURE: Expose 'max torrent file size' setting (Chocobo1) 66dfe8545dff47830fdc5c118c7e11d2cad21975
+    - FEATURE: Expose 'bdecode limits' settings (Chocobo1) 5a660fc8a979aa6b97c9059df4216da8b9a80a57
+    - FEATURE: Add options to adjust behavior of merging trackers to existing torrent (glassez) 4ef9a6444ae9067bb40fd2f5442564416e047689
+    - FEATURE: Add option to stop seeding when torrent has been inactive (Christopher) 35e18498d9bde2292f4c0be793f0870385e54770
+    - FEATURE: Allow to use proxy per subsystem (glassez) 6ac14d0c571995b3a3b750b9afdd6ff4c34797e9
+    - FEATURE: Allow to globally disable the use of proxy (glassez) 7ec80263e14a340395bf60f7da78c2cadd134b31
+    - FEATURE: Expand the scope of "Proxy hostname lookup" option (glassez) 96da685e5d72061cab7caf4d68a9e2c8dea86de7
+    - FEATURE: Add shortcut for "Ban peer permanently" function (Luka Čelebić) 917190d936c0287d8d5e5c3956e7f936c8c90bb6
+    - FEATURE: Add option to auto hide zero status filters (glassez) 0dcb65bb7cc2f4f65405712d6159d429abb5a3a5
+    - FEATURE: Allow to disable confirmation of Pause/Resume All (glassez) cf415dd7fed4b34abe7cd6aa4e68089cfdc724e6
+    - FEATURE: Add alternative shortcut CTRL+E for CTRL+F (Luka Čelebić) 379b0dbe40a75e9d7f5ec4f638dbadbb78cef206
+    - FEATURE: Show filtered port numbers in logs (Hanabishi) d75fd3fcde8d4cc1c7ddb1ebeaf56791ddd5e3c8
+    - BUGFIX: Ensure ongoing storage moving job will be completed when shutting down (Chocobo1) 5f2d80786132eaf0569fa75e4937aafc57b3f302
+    - BUGFIX: Refactored many areas to call non UI blocking code (glassez)
+    - BUGFIX: Various improvements to the SQLite backend (glassez)
+    - BUGFIX: Improve startup window state handling (glassez) 32e437120823120e233ea589c3901cfc9b528f82
+    - BUGFIX: Reduce default file pool size (glassez) 8bcac1bed28f93c0712e915f43294b1e5fd03659
```

Comment:
It is changed again in BUGFIX: Change default value for `file pool size` and `stop tracker timeout` settings (stalkerok) 488464731d1271f2ecd435beb3c956e7a1958bc5
So you don't need to mention it twice.